patches/0013-p256-armv8-optimizations

```diff
@@ -2343,15 +2352,43 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256-x86_64.c
      return ec_simple_scalar_to_montgomery_inv_vartime(group, out, in);
    }
 +#endif
- 
+
    assert(group->order.width == P256_LIMBS);
    if (!beeu_mod_inverse_vartime(out->words, in->words, group->order.d)) {
 @@ -629,5 +631,6 @@ DEFINE_METHOD_FUNCTION(EC_METHOD, EC_GFp
    out->cmp_x_coordinate = ecp_nistz256_cmp_x_coordinate;
  }
- 
+
 -#endif /* !defined(OPENSSL_NO_ASM) && defined(OPENSSL_X86_64) && \
--          !defined(OPENSSL_SMALL) */
-+#endif /* !defined(OPENSSL_NO_ASM) &&                           \
-+          (defined(OPENSSL_X86_64) || defined(OPENSSL_AARCH64)) &&  \
-+          !defined(OPENSSL_SMALL)*/
++#endif /* !defined(OPENSSL_NO_ASM) && \
++          (defined(OPENSSL_X86_64) || (defined(OPENSSL_AARCH64_P256) && defined(OPENSSL_AARCH64))) &&  \
+           !defined(OPENSSL_SMALL) */
+Index: aws-lc/third_party/boringssl/CMakeLists.txt
+===================================================================
+--- aws-lc.orig/third_party/boringssl/CMakeLists.txt
++++ aws-lc/third_party/boringssl/CMakeLists.txt
+@@ -477,6 +477,10 @@ if(OPENSSL_NO_SSE2_FOR_TESTING)
+   add_definitions(-DOPENSSL_NO_SSE2_FOR_TESTING)
+ endif()
+
++if(OPENSSL_AARCH64_P256)
++  add_definitions(-DOPENSSL_AARCH64_P256)
++endif()
++
+ if(OPENSSL_NO_ASM)
+   add_definitions(-DOPENSSL_NO_ASM)
+   set(ARCH "generic")
+Index: aws-lc/tests/ci/run_posix_tests.sh
+===================================================================
+--- aws-lc.orig/tests/ci/run_posix_tests.sh
++++ aws-lc/tests/ci/run_posix_tests.sh
+@@ -16,6 +16,9 @@ build_and_test -DOPENSSL_SMALL=1 -DCMAKE
+ echo "Testing AWS-LC in no asm mode."
+ build_and_test -DOPENSSL_NO_ASM=1 -DCMAKE_BUILD_TYPE=Release
+ 
++echo "Testing AWS-LC with AArch64 P-256 optimizations."
++build_and_test -DOPENSSL_AARCH64_P256=1 -DCMAKE_BUILD_TYPE=Release
++
+ echo "Testing building shared lib."
+ run_build -DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release
+ 
```

Comment:
This change does not need to be made in a patch since it is outside of the `third_party` directory

Reply:
That's a good point. I guess I can remove it and postpone adding the CI test until after I merge main and remove quilt.